testeng/views/testengViews.groovy

```diff
@@ -0,0 +1,17 @@
+package testeng
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.DEFAULT_VIEW
+
+listView("upgrade-python-requirements") {
```

Comment:
Thanks for thinking of this, this should come in handy.  We should also prepare a GitHub search URL that returns all open python dependency upgrade PRs across repos in the edX organization.